.github/workflows/terraform-fmt.yml

```diff
@@ -11,16 +9,14 @@ jobs:
     steps:
       # checkout code
       - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
 
       # format terraform
       - name: Setup Terraform
         uses: hashicorp/setup-terraform@v2
         with:
           terraform_version: "1.3.0"
       - name: Terraform fmt
-        run: terraform fmt -recursive
-
-      # Commit all changed files back to the repository
-      - uses: stefanzweifel/git-auto-commit-action@3ea6ae190baf489ba007f7c92608f33ce20ef04a
```

Comment:
This is problematic with forks as we don't have correct permissions to push to those. Have removed committing entirely and added `-check` as this seems to be the pattern we follow elsewhere for format actions in CI.